string ee_name
---
bool success
string message